### DISCUSSION TITLE
move devices.json

### DISCUSSION THREAD

**Discussion by liput** [jasonacox/tinytuya#418] [General]
How can I move the devices.json file and specify a new location?

**Comment by uzlonewolf** [comment]
I've not tried it, but this should work:
```python
import tinytuya

tinytuya.DEVICEFILE = '/path/to/devices.json'

d = tinytuya.Device(...)
```

**Comment by liput** [comment]
I found another option before your reply after looking more in the source code

from tinytuya import scanner
f = open('/path/devices.json')
tuyadevices = json.load(f)
devices = scanner.devices(tuyadevices=tuyadevices, poll=True)

thanks
